polyfills/Array/prototype/@@iterator/config.json

```diff
@@ -0,0 +1,13 @@
+{
+	"browsers": {
+		"chrome": "* - 38",
+		"firefox": "* - 36",
+		"ie": "* - 11",
+		"ie_mob": "10 - *",
+		"opera": "* - 25",
+		"safari": "* - 8"
```

Comment:
A good pattern I've been using is `<`, because then you can list the version number the feature was added in, rather than the one before that, and it makes like easier when comparing our configs to MDN, caniuse etc.